LEGO1/lego/legoomni/src/actors/skateboard.cpp

```diff
@@ -37,11 +67,16 @@ MxU32 SkateBoard::VTable0xcc()
 	return 0;
 }
 
-// STUB: LEGO1 0x10010230
+// FUNCTION: LEGO1 0x10010230
 MxU32 SkateBoard::VTable0xd4(LegoControlManagerEvent& p_param)
 {
-	// TODO
-	return 0;
+	MxU32 result = 0;
+	if (p_param.m_unk0x28 == 1 && p_param.m_clickedObjectId == 0xc3) {
```

Comment:
This should use the getters on the class instead (see other comment); the hex value is almost certainly also a constant in the script action enum header